README.md

```diff
@@ -138,6 +153,16 @@ Spark Version | Release # | Binary Location
      df.filter(df("airportCode") >= "CAA").select("airportCode","airportName").show()
      df.filter(df("airportCode") >= "CAA").select("airportCode","airportName").write.format("com.cloudant.spark").save("airportcodemapping_df")
      
+In case of doing multiple operations on a dataframe (select, filter etc.), you should persist a dataframe. Othewise, every operation on a dataframe will load the same data from Cloudant again. Persisting will also speed up computation.  This will persist an RDD in memory: 
+
+	df.cache() 
+
+Alternatively for large dbs to persist in memory & disk, you can use:
+
+	import org.apache.spark.storage.StorageLevel
+	....
+	df.persist(StorageLevel.MEMORY_AND_DISK)     
+     
 
```

Comment:
I'm not a big fan of having the text duplicated. Let's again improve the code snippet itself (putting the df.cache() and df.persist() calls inline after the load and drop the rest please. The explanation is already in the section above and the import statement is in the scala code sample. Makes sense?